djangocms_icon/fields.py

```diff
@@ -5,13 +5,24 @@
 from django.db import models
 from django.forms import fields, widgets
 from django.utils.translation import ugettext
+from django.template.loader import render_to_string
 
 
 def get_iconsets():
-    return getattr(settings, 'DJANGOCMS_ICON_SETS', (
-        ('fontawesome', 'fa', 'Font Awesome'),
+    iconsets = getattr(settings, 'DJANGOCMS_ICON_SETS', (
+        ('fontawesome5regular', 'far', 'Font Awesome 5 Regular', 'lastest'),
+        ('fontawesome5solid', 'fas', 'Font Awesome 5 Solid', 'lastest'),
+        ('fontawesome5brands', 'fab', 'Font Awesome 5 Brands', 'lastest'),
     ))
 
+    set = []
+    for iconset in iconsets:
+        if len(iconset) == 3:
```

Comment:
I'm not sure about the addition of this code here, is it specifically for fontawesome5?

Reply:
This is to keep backward compatibility if no icon set version is provided in the settings.py. 
It also allows to use the latest version provided by iconset.
This change is not specific to fontawesome 5, it can be used with all iconsets.